files/tasks/packit-service-requirements.yaml

```diff
@@ -3,6 +3,7 @@
   ansible.builtin.dnf:
     name:
       - ansible
+      - ansible-collection-community-general
```

Comment:
A comment about removing it once we move [the base job](https://github.com/packit/packit-service-zuul/blob/main/zuul.d/jobs.yaml#L20) to F36, would be nice.